SPECS/curl/CVE-2023-38546.patch

```diff
@@ -1,125 +0,0 @@
-From a3aebb3421fd48dcb4f48743109dc0c4c82f8916 Mon Sep 17 00:00:00 2001
```

Comment:
Fix present in 8.5.0: https://github.com/curl/curl/commit/61275672b46d9abb3285740467b882e22ed75da8